src/Product/WCProductAdapter.php

```diff
@@ -109,6 +109,7 @@ public function mapTypes( $properties ) {
 		$this->map_woocommerce_product();
 		$this->map_attribute_mapping_rules( $mapping_rules );
 		$this->map_gla_attributes( $gla_attributes );
+		$this->map_gtin();
```

Comment:
Reading this, I wanted to point out that it won't be allowed to override the GTIN via Google for WooCommerce tab or Attribute Mapping rules. Just wanted to be sure that this is expected.